guides/common/modules/con_web-ui-overview.adoc

```diff
@@ -4,7 +4,32 @@
 = {ProjectWebUI} overview
 
 You can manage and monitor your {Project} infrastructure from a browser with the {ProjectWebUI}.
-For example, you can use the following navigation features in the {ProjectWebUI}:
+ifdef::satellite[]
+{Project} supports recent versions of the Mozilla Firefox and Google Chrome browsers.
+endif::[]
+ifdef::foreman-el,foreman-deb,katello[]
+{Team} recommends the following major browsers:
+
+* Apple Safari
+* Google Chrome
+* Microsoft Edge
+* Mozilla Firefox
+* Mozilla Firefox Extended Support Release (ESR)
+
+[NOTE]
+====
+Use the latest versions of a major browser.
+Other browsers and their older versions might work unpredictably.
+====
+endif::[]
+
+ifdef::satellite[]
+The {ProjectWebUI} and command-line interface support English, French, Japanese, Korean, and Simplified Chinese.
+endif::[]
+ifndef::satellite[]
+The {ProjectWebUI} and command-line interface is translated into various languages.
+endif::[]
+The navigation features in the {ProjectWebUI} include the following:
 
 [cols="2,7", options="header"]
```

Comment:
Any thoughts on potentially replacing the list of navigation features with an actual screenshot? I think it would be more informative and helpful. (The way I see it, the point here is to showcase the web UI to potential new users. Seeing a screenshot would do that better than listing terms that might still be unknown to users at this point.)

Reply:
I am OK with adding a screenshot. Though the screenshot would lack the description and users cannot search for it.

Reply:
Screenshots added.

> Can we keep the keywords close to the screenshot in a hidden way so that they're indexed for search, but not visible in the build? 🤔 Just a thought.

What do you mean?

Reply:
Okay! Hm, looking at the lines I'm replacing with the screenshot, they are a list of features like Hosts, Configure, Infrastructure, notifications, etc. I don't consider these to be the key of what users would be searching this section for. The section is centered around "web UI" and I think the text covers that well. No action.